src/bitmessagemain.py

```diff
@@ -183,7 +190,8 @@ def signal_handler(signum, frame):
               ' because the UI captures the signal.')
 
 
-class Main:
+class Main(object):
```

Comment:
@g1itch @PeterSurda 
I tested and work fine.
Is it okay to use object here?

Reply:
Should be OK. I don't understand the reason of using old classes at all. Maybe it was a sort of optimization. If you don't inherit from both old and new style classes it's the right way to use new style classes in python 2.7.

Reply:
I also think it's ok.